README.md

```diff
@@ -9,38 +9,35 @@ Our company is building a prototype ticketing system for a local burrito shop. I
 * Run `npm install` to install dependencies
 * Run `npm start` to start your development server
 
-Be sure to setup the backend repo for Burrito Builder to be able to retrieve and save burrito orders.
+Be sure to setup the [backend repo for Burrito Builder](https://github.com/turingschool-examples/burrito-builder-api) to be able to retrieve and save burrito orders.
 
-## Iterations
+## Iterations (new)
 
 ### Iteration 1
 
-Right now the `App` component is connected to the store. While sometimes it can be good to have `App` know about everything, in this case the developer team said that it's not necessary.
-
-Remove Redux from the `App` component and move that functionality to the `Orders` component. Make `App` into a functional component, and bring in the burrito orders directly into the `Orders` component instead.
+Right now the app has a bug somewhere in it. Run the dev server and fix whatever bug is causing the app to crash. 
 
 ### Iteration 2
 
-Add functionality to allow submission of the form when there is at least one ingredient added to the order. If there are no ingredients in the order, you should not be able to submit the order.
-
-Also, right now we're seeing a warning in the console about unique keys... Get rid of that warning.
+Connect the front end to the backend so that when the app loads all existing orders from the server.
```

Comment:
so that when the app loads, all existing orders are fetched from the server

Reply:
Fixed!